impeller/golden_tests_harvester/pubspec.yaml

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: golden_tests_harvester
```

Comment:
Please add an entry to https://github.com/flutter/engine/blob/main/tools/pub_get_offline.py#L22, which will make sure that all of the pub dependencies are prefetched.

Reply:
done